src/Context.js

```diff
@@ -0,0 +1,9 @@
+import React, { createContext, useState } from 'react';
+
+export const Context = createContext();
+
+const ContextProvidor = (props) => {
+  const { user } = useState('Guest');
+  return <ContextProvidor value={{ user }}>{props.children}</ContextProvidor>;
```

Comment:
this line should be like this:


```suggestion
  return <Context.Provider value={{ user }}>{props.children}</Context.Provider>;
```

because it is a special component!